js/example-frontend-js/src/ExampleMain.kt

```diff
@@ -35,8 +36,7 @@ private fun HTMLElement.setPosition(x: Double, y: Double) {
     }
 }
 
-@Suppress("DEPRECATION")
-private fun random() = kotlin.js.Math.random()
+private fun random():Double = Random.nextDouble()
```

Comment:
One empty line should also be removed.

Reply:
Which one? I don't add empty lines :)

Reply:
Sorry. My bad. Disregard.

Reply:
I somehow thought I've removed the function completely. Actually, there's now no much sense in having this function in the first place -- it would be better if it is just inlined to all its use-sites.

Reply:
Inlined